clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -418,7 +418,6 @@ export const CustomTextArea = ({
     <Textarea
       {...field}
       size="sm"
-      width="auto"
```

Comment:
The two other components I saw using `CustomTextArea` were `TaxonomyFormBase` and `DatasetYamlForm`. I double checked both forms on main and this branch to make sure removing this field didn't break anything and everything looked good 👍 


The pages were `/dataset/new` and `/taxonomy`